content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -383,6 +383,12 @@ Run 'kubectl get nodes' on control-plane to see this machine join.
 A few seconds later, you should notice this node in the output from `kubectl get
 nodes` when run on the control-plane node.
 
+{{< note >}}
+As the cluster nodes are usually initialized sequentially, the CoreDNS Pods are likely to 
+run on the first control-plane node entirely. If you install a cluster with more than one node, 
```

Comment:
“entirely” in this context reads strange.

Reply:
Thanks @tengqm 
Updated!